apps/notepad/src/app/component/notes-list/notes-list.component.html

```diff
@@ -0,0 +1,32 @@
+<form [formGroup]="searchForm" (ngSubmit)="onSearchSubmit()">
```

Comment:
why are you using form group form group for one form control? you can use FormControl instead